core/dbt/config/project.py

```diff
@@ -401,6 +402,7 @@ def from_project_config(
         )
 
         docs_paths: List[str] = value_or(cfg.docs_paths, all_source_paths)
+        asset_paths: List[str] = value_or(cfg.asset_paths, ['assets'])
```

Comment:
heads up, this differs from the original ticket that had the default as `['static']`, but I prefer the consistency